coremltools/converters/mil/frontend/_utils.py

```diff
@@ -294,15 +296,25 @@ def solve_sum_einsum_one_step(src_axes, used_by_other_axes, x):
     return ret_parsed_vectors, vars
 
 
-def solve_generic_einsum(parsed_vectors, a_var, b_var, name):
+def get_perm_transpose_einsum(src_axes, dst_axes):
```

Comment:
Please add type annotation to these parameters. Looks like you basically have that in the docstring already.